emacs-everywhere.el

```diff
@@ -85,7 +85,7 @@ To not run any command, set to nil."
   (pcase (car emacs-everywhere--display-server)
     ('x11 (list "xclip" "-selection" "clipboard" "%f"))
     ('wayland
-     (list "sh" "-c" "wl-copy < %f"))
+     (list "sh" "-c" "wl-copy -p < %f"))
```

Comment:
Not sure if this will work on KDE. I think by default KDE binds Shift+Insert to pasting from clipboard buffer rather than primary (which is not standard). Try removing this is if it doesn't work and let me know.